keyboards/minidox/keymaps/norman/keymap.c

```diff
@@ -0,0 +1,70 @@
+#include QMK_KEYBOARD_H
+
+#include "keymap_norman.h"
+
+#define PERMISSIVE_HOLD
```

Comment:
Lines 5-7 here should all be in `keyboards/minidox/keymaps/norman/config.h` instead.

```suggestion
```